specification/subscription/resource-manager/readme.md

```diff
@@ -19,42 +19,14 @@ To see additional help and options, run:
 
 ## Configuration
 
-
-
 ### Basic Information 
 These are the global settings for the Subscription API.
 
-``` yaml
-openapi-type: arm
-tag: package-2018-03-01-preview
-```
-
 ``` yaml
 openapi-type: arm
 tag: package-2017-11-preview
 ```
 
-### Tag: package-2018-03-01-preview
-
-These settings apply only when `--tag=package-2018-03-01-preview` is specified on the command line.
-
-``` yaml $(tag) == 'package-2018-03-01-preview'
-input-file:
-- Microsoft.Subscription/preview/2018-03-01-preview/subscription.json
-```
-
-### Tag: package-all-subscription
```

Comment:
there's more references to package-all-subscription remaining in this file - please remove all